### ISSUE TITLE
False positive with SASS numeric operators.

### ISSUE THREAD

**Issue by dimascyriaco** [AndyOGo/stylelint-declaration-strict-value#135]
Since updating to 1.7.7 (was on 1.6.0) I'm getting this error:

```
Expected variable, function or keyword for "*" of "width"    scale-unlimited/declaration-strict-value
```

In this rule:

```css
width: $spacer * 2;
```

The plugin is configured like this:

```json
"scale-unlimited/declaration-strict-value": [
  ["width", "height"],
  {
    "ignoreKeywords": {
      "": ["inherit", "100%", "50%", "0", "auto"],
    }
  }
]

```

It was working fine previously.

**Comment by AndyOGo** [comment]
Thank you.
Though that was actually a bug previously, since `$spacer * 2` is not a variable alone, it's an expression derived from a multiplication.

**Comment by dimascyriaco** [comment]
Ok. Thank you.
